WordPress/Classes/ViewRelated/Views/AutoResizingLabel.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+class AutoResizingLabel: UILabel {
+    override func intrinsicContentSize() -> CGSize {
+        var size = super.intrinsicContentSize()
+        
+        // This, unfortunately, fixes an issue seen in the plans feature list's self-sizing cells
+        // where only the first line of text would display on the iPhone 6(s) Plus.
+        if numberOfLines == 0 {
```

Comment:
Mentioned on slack but leaving it here for reference: setting the stack view distribution to "Fill Proportionally" seems to work as well, so we wouldn't need this custom class